examples/repl/repl.cpp

```diff
@@ -0,0 +1,60 @@
+#include <kalcy/kalcy.hpp>
+#include <cassert>
+#include <format>
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+
+namespace {
+struct Repl {
+	const std::string_view EXIT_TOKEN{"exit"};
+	bool verbose{}; // toggled on with '-v' and off with '-q'
+	bool is_running{true};
+
+	auto start() -> bool {
+		while (is_running) {
+			std::string text{};
+			std::cout << std::format("[{}] > ", verbose ? "verbose" : "quiet");
+			std::getline(std::cin, text);
+			// run kalcy on input expression
+			if (!run(text)) { return EXIT_FAILURE; }
+		}
+		// print epilogue
+		std::cout << std::format("\n^^ kalcy v{}\n", kalcy::version_v);
+		return EXIT_SUCCESS;
+	}
+
+	auto run(std::string_view const text) -> bool {
```

Comment:
Reduce nesting in this function. Leverage early returns to avoid `else if` / `else`.